$OpenBSD: patch-GNUmakefile,v 1.11 2008/10/18 18:58:06 kili Exp $

Don't rebuild the documentation, and don't install the pdf files
contained in the distfile.
This is required in addition to `--without-manual' in CONFIGURE_ARGS.

--- GNUmakefile.orig	Thu Oct  2 13:47:19 2008
+++ GNUmakefile	Thu Oct  2 20:10:58 2008
@@ -191,10 +191,9 @@ installbin:	all
 		$(INSTALL) -d $(DESTDIR)$(mandir)/man1
 	$(INSTALL_DATA) doc/darcs.1 $(DESTDIR)$(mandir)/man1/
 
-installdocs:	doc/manual/darcs.pdf doc/manual/patch-theory.pdf doc/manual/index.html
+installdocs:
 	test -d $(DESTDIR)$(docdir)/manual || \
 		$(INSTALL) -d $(DESTDIR)$(docdir)/manual
-	$(INSTALL_DATA) doc/manual/*.pdf $(DESTDIR)$(docdir)/manual
 	$(INSTALL_DATA) doc/manual/*.png $(DESTDIR)$(docdir)/manual
 	$(INSTALL_DATA) doc/manual/*.html $(DESTDIR)$(docdir)/manual
 
